Fixes to merge (#13)
eficode-academy/rf-advanced-web-katas: exercises/08-execute-javascript.md

<details>
  <summary>Browser</summary>

In contrast to SeleniumLibrary's implementation, Browser's keyword to execute JavaScript takes two arguments. The second argument is
a reference to an element in the web page, and the first one is the JS function,
that will be sent to that referenced element. Locator is not the same as a reference, so we will need to call another keyword to create a reference first.

The operation on the slider is thus not a one-liner, so let's make a
separate keyword for it, which can be then called by `Run Keyword If`.
The new keyword should contain the steps to get a reference for the slider, and change its value with a simple JS function.

- Add a keyword `Change slider value with JS` that will change the slider's value to `wanted_value`.

Reference to an element is returned by `Get Element` keyword, which takes a
locator as an argument. Store it in a local variable. It can be used like a
locator in other keywords, even without the need to specify the iframe.
With this direct reference to the slider, JS function needed is a simple change of the `value` property.
Example function syntax: `(element) => element.property = "new value"`.

> - Get and store slider's reference, and use it to assign `wanted_value` to slider's `value` property.

Reviewer: I would give more bulleted tasks for clarity:
```suggestion
- Use `Get Element` to get a reference to the slider element
- Use `Execute Javascript` to assign `wanted_value` to the slider's `value` property using the reference element.
```

> Keyword should also check that the value stored by the slider indeed changed.

Reviewer: ```suggestion
Our keyword should also check that the value stored by the slider indeed changed.
```

Author: Suggestion implemented.

This can be done by confirming the value before the change is `0`, and after is
equal to the `wanted value`. Browser's keywords have built-in checking, so
there's no need to store the numbers and compare them as integers. The value is stored inside slider's properly, so you can use the `Get Property` keyword.

> - Implement checks with `Get Property` to ensure the value has been changed.

Reviewer: Keyword validation is explained later on in the exercise file, but it's used here for the first time, so it should be explained here:

```suggestion
Browser library has builtin waiting and no separate validation keywords. Instead, you can use `Get Text`
to validate the text automatically. Python validations such as `==`, `!=`, `contains`, and `not contains`
are available for validation. The syntax is Python-esque:

```robot
Get Property    locator    ==    some value
```

- Implement checks with `Get Property` to ensure the value has been changed.
```

Author: Code blocked moved here.

Browser library has builtin waiting and no separate validation keywords. Instead, you can use `Get Text`
to validate the text automatically. Python validations such as `==`, `!=`, `contains`, and `not contains`

> are available for validation. The syntax is Python-esque:

Reviewer: Since Browser library evaluations should be explained earlier (see previous comment), this paragraph should be moved there.
